### PR TITLE
Atalay/23 fill the database with data

### PR DESCRIPTION
**Pull request by Atalay-Polat**
Added answers and questions in seed()

### CODE REVIEW

File: InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs [hunk 1]
```diff
@@ -1,20 +1,148 @@
...
 using Microsoft.AspNetCore.Identity;
 using Portal.Core.Constants;
+using Shared.Enums;
+using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
 
 namespace ExaLearn.Dal
 {
     public static class DbInitializer
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs:12]: image
move

Comment (romanstat) [reply InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs:12]: when you execute update-database db should be filled

File: InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs [hunk 2]
```diff
@@ -1,20 +1,148 @@
...
+            {
+            new Question
+            {
+                Type = QuestionType.Essay,
+                Level = LevelType.Beginner,
+                Description = "Essay description",
+                Score = 10, //change according to requirements
+                Text = "Theme: Global warming",
+                AudioId = null,
+                //Answer // to be considered
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Dal/Database/DbInitializer.cs:108]: add answer to list

File: InternshipBE/ExaLearn.Dal/Entities/Question.cs [hunk 3]
```diff
@@ -8,7 +8,7 @@ public class Question
     {
         public int Id { get; set; }
 
-        public int AudioId { get; set; }
+        public int? AudioId { get; set; }
```
Comment (romanstat) [inline InternshipBE/ExaLearn.Dal/Entities/Question.cs:11]: for what?

File: InternshipBE/ExaLearn.WebApi/Controllers/DatabaseController.cs [hunk 4]
```diff
@@ -10,19 +11,27 @@ namespace ExaLearn.WebApi.Controllers
     [ApiController]
     public class DatabaseController : ControllerBase
```
Comment (romanstat) [inline InternshipBE/ExaLearn.WebApi/Controllers/DatabaseController.cs:12]: n-layer architecture! DAL BL PRESENTATION
Remove repository from Presentation layer

File: InternshipBE/ExaLearn.Dal/Entities/Question.cs (hunk 3 above)
Comment (Atalay-Polat) [reply InternshipBE/ExaLearn.Dal/Entities/Question.cs:11]: > for what?
Because only listening questions have audio file in.

**Pull request closed without merging**